docs/ossm/create-mesh/README.md

```diff
@@ -1,65 +1,34 @@
 
-Adding your workload to a mesh is a two steps process:
-1. Your workload must be discovered by Istio control plane
-1. Your workload must be injected with Istio Proxy sidecar
+A service mesh will include a workload that:
+1. Has been discovered by the service mesh control plane
+1. Has been [injected with a Envoy proxy sidecar](../injection/README.md)
 
-First step is described in this guide, see [Installing the Sidecar
-](../injection/README.md) for second step.
-
-## Concepts
-To better understand next chaptes, it's recommended to read Service mesh architecture guide first.
-
-### Cluster wide by default
-In order to program the service mesh, the Istio control plane (Istiod) reads a variety of configurations, including core OpenShift types like Service and Node, and Istio’s own types like Gateway. These are then sent to the data plane.
-
-By default, the control plane will read all configuration in all namespaces. Each proxy instance will receive configuration for all namespaces as well. This includes information about workloads that are not enrolled in the mesh.
-
-This default ensures correct behavior out of the box, but comes with a scalability cost. Each configuration has a cost (in CPU and memory, primarily) to maintain and keep up to date. At large scales, it is critical to limit the configuration scope to avoid excessive resource consumption.
-
-### Soft multi-tenancy
-Is defined as having a single OpenShift control plane with multiple Istio control planes and multiple meshes, one control plane and one mesh per tenant. The cluster administrator gets control and visibility across all the Istio control planes, while the tenant administrator only gets control of a specific Istio instance. Separation between the tenants is provided by OpenShift namespaces and RBAC.
-
-## Configuration scoping
-Istio offers a few tools to help control the scope of a configuration to meet different use cases. Depending on your requirements, these can be used alone or together.
-
-- `Sidecar` provides a mechanism for specific workloads to import a set of configurations
-- `exportTo` provides a mechanism to export a configuration to a set of workloads
-- `discoverySelectors` provides a mechanism to let Istio completely ignore a set of configurations
-
-This chapter is focusing on `DiscoverySelectors`, see [Single-mesh isolation ("zone") features](../zones/README.md) for details about `Sidecar` and `exportTo`.
+By default, the service mesh control plane will watch all namespaces within the cluster, meaning that any workload with the appropriate sidecar or namespace injection label will be added to the service mesh. This may not be desirable in a shared cluster, and you may want to limit the scope of the service mesh to only a portion of your cluster. This is particularly important if you plan to have [multiple service meshes within the same cluster](./multi-control-planes/README.md).
```

Comment:
"By default, the service mesh control plane will watch all namespaces within the cluster, meaning that any workload with the appropriate sidecar or namespace injection label will be added to the service mesh."

This might be a bit confusing. The problem is that by default it discovers resources in namespaces even without any labels and send them to all proxies in the mesh. I will include a bit more detailed description.